### PR TITLE
Add volume iops

### PR DESCRIPTION
**Pull request by krallin**
This adds volume IOPS in the Dashboard for databases. I also included a bit of refactoring of the metrics components.

cc @fancyremarker @gib @sandersonet

### CODE REVIEW

File: app/components/container-iops-metrics/component.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+              if (column[j] > axisMax) {
+                axisMax = column[j];
+              }
+            }
+          }
+        }
+      }
+
+      // Finally, we add some padding here to avoid craming the layout.
+      return Math.round(axisMax * 1.1);
```
Comment (gib) [inline app/components/container-iops-metrics/component.js]: Thank you for these comments. very clear

File: app/mixins/components/container-metrics.js [hunk 2]
```diff
@@ -36,46 +36,55 @@ export default Ember.Mixin.create({
...
-    if (axisMax) {
-      axis.y.max = axisMax;
-    }
+      if (h.axisMax) {
+        axis.y.max = h.axisMax;
+      }
 
-    return axis;
+      return axis;
+    });
```
Comment (gib) [inline app/mixins/components/container-metrics.js]: Nice Promise work

**Comment by krallin** [comment]
@fancyremarker : please have a look at the description when you have a minute. Perhaps we should show the DB volume type to give customers a little more visibility?

**Comment by sandersonet** [comment]
:+1: Perfect!

**Comment by fancyremarker** [comment]
@krallin: As discussed, description looks good as-is. Also fine with showing the database volume type (and/or adding additional explanation related to volume types) — just a bit unsure about how to do this in a sufficiently concise way.

**Comment by krallin** [comment]
@fancyremarker Thanks! I'll ship as-is and look into whether there's a good option to show the volume type as well.

**Pull request merged**